kmp-library/src/jvmMain/kotlin/uk/co/lucasweb/aws/v4/signer/hash/Sha256ToHexString.kt

```diff
@@ -0,0 +1,7 @@
+package uk.co.lucasweb.aws.v4.signer.hash
+
+object Sha256ToHexString {
```

Comment:
This helper needs to live in the `jvm` module since `String.toByteArray()` doesn't exist on any other platform then the `jvm`. This is actually a blocker we need to have a look at.

Reply:
Luckily I found an [extension function for kotlin/native](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.native/to-utf8.html) that does the same.
00b920a